docs/themes.md

```diff
@@ -24,3 +24,25 @@ When a [new site](docs://guides/multiple-sites.md) is created, a "default" theme
 	}
 }
 ```
+
+## Network Enabled Themes
+
+By default all themes are network enabled, this means that every theme in your application is available to all sites on the network.
+
+Individual themes can still be disabled for the network, and enabled for specific sites by editing a site in the network admin.
+
+If you prefer to use the standard WordPress behaviour where themes are disabled for the network by default you can set the `network-enable-themes` config option to `false`:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"network-enable-themes": false
```

Comment:
I wonder if something like `enable-all-themes` might be clearer?

Reply:
I'm not sure there is a clear name for it given that you subsequently disable them. Nothing is being set in stone so it doesn't really feel like a config option is actually a good fit to be honest.

Reply:
hmm, yeah; `enable-new-themes-by-default` which is a bit long winded 🤔